lib/src/widgets/chat_groupedlist_widget.dart

```diff
@@ -207,6 +222,13 @@ class _ChatGroupedListWidgetState extends State<ChatGroupedListWidget>
                                 onSwipe: widget.assignReplyMessage,
                                 receiver: widget.receiver,
                                 sender: widget.sender,
+                                shouldHighlight: _replyId == message.id,
+                                onReplyTap: widget.repliedMessageConfig
+                                            ?.enableScrollToRepliedMsg ==
+                                        true
+                                    ? (replyId) =>
```

Comment:
Put only widget.repliedMessageConfig?.enableScrollToRepliedMsg instead comparing bool

Reply:
^^

Reply:
you can use null aware operator with false value.